Kickstarter-iOS/Views/Controllers/SignupViewController.swift

```diff
@@ -120,8 +120,16 @@ internal final class SignupViewController: UIViewController, MFMailComposeViewCo
 
     self.viewModel.outputs.logIntoEnvironment
       .observeValues { [weak self] in
```

Comment:
This didn't throw an error about the main thread despite not having `observeForControllerAction()` or `observeForUI()` so I left it as it is. Should I add one of those in for good measure?

Reply:
We would use it here too, it's possible that this signal emits on the main thread but we want to always be sure that it does so we should just use `observeForControllerAction()` to not take any chances.